src/Loadbalancers/Entities/Action.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace UKFast\SDK\Loadbalancers\Entities;
+
+use UKFast\SDK\Entity;
+
+/**
+ * @property string $id
+ * @property string $function
+ * @property string $firstArg
+ * @property string $secondArg
```

Comment:
```suggestion
 * @property string $secondArg
 * @property string $thirdArg
```
Missing third arguement in the comments.This is the response that I have recieved from the sdk
```
[
      "id" => "ad882009-e9dc-43dd-a87f-1c25e51b262e"
      "function" => "http-request"
      "firstArg" => "test"
      "secondArg" => null
      "thirdArg" => null
      "conditional" => null
      "index" => 0
    ]
```